mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -17,6 +18,15 @@ interface LocalizedStrings {
   val authenticateSwitchToRegister: String
   val authenticateErrorFailure: String
 
+  /** Profile Strings **/
+  val profileMatchTitle: (String) -> String
+  val profileMatchInfo: (MatchResult, MatchResult.Reason, Int) -> String
```

Comment:
It looks a bit weird to me that some "feature" types are used in the `LocalizedStrings` interface. Maybe the `profileMatchInfo` tries to do too much stuff, and we should have more specialised strings such as `profileMatchInfoWin`, `profileMatchInfoDraw`, `profileMatchInfoLoss`, etc.